wrappers/javascript/aries-askar-react-native/src/index.ts

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
I added them because I was having some errors and wanted to match the index from indy-vdr-react-native. Seems to not be needed so I'm reverting to original code.